legal-api/tests/unit/models/test_legal_entity.py

```diff
@@ -776,18 +753,16 @@ def test_business_name(session, entity_type, legal_name, operating_name, expecte
             # expected_alternate_names
             [
                 {
-                    "entityType": "SP",
                     "identifier": ALTERNATE_NAME_1_IDENTIFIER,
-                    "nameRegisteredDate": ALTERNATE_NAME_1_REGISTERED_DATE,
-                    "nameStartDate": ALTERNATE_NAME_1_START_DATE,
                     "operatingName": ALTERNATE_NAME_1,
+                    "entityType": "SP",
+                    "nameRegisteredDate": ALTERNATE_NAME_1_START_DATE_ISO,
```

Comment:
These need to be in a specific order to compare them with `assert` down below. Also, the comparison is done to the `json` of `AlternateName`, which does not return `nameStartDate`. `nameRegisteredDate` is returned in the json as the `start_date` property.